README.md

```diff
@@ -55,10 +52,10 @@ Release versions follow the [semver](https://semver.org/) convention of incompat
 
 ### Dependencies
 
-- [Ginkgo](https://onsi.github.io/ginkgo/#installing-ginkgo) for running the end-to-end tests
-- Docker and Docker Compose v2 for running the local test network
+- [Ginkgo](https://onsi.github.io/ginkgo/#installing-ginkgo) for running the end-to-end tests.
+- Docker and Docker Compose v2 for running the local test network.
   - The docker image installs the following:
-    - [Foundry](https://book.getfoundry.sh/getting-started/installation)
+    - [Foundry](https://book.getfoundry.sh/) Use `/scripts/install_foundry.sh` to install the ava-labs fork.
```

Comment:
```suggestion
    - [Foundry](https://book.getfoundry.sh/) Use `/scripts/install_foundry.sh` to install the Ava Labs [fork](https://github.com/ava-labs/foundry).
```

Capitalize Ava Labs, or put `ava-labs` in backticks to emphasize you're referring to the Github entity.